ci-operator/config/shiftstack/bugwatcher/shiftstack-bugwatcher-main.yaml

```diff
@@ -38,6 +39,11 @@ tests:
   steps:
     test:
     - ref: shiftstack-bugwatcher-pretriage
+- as: bugwatcher-triage
+  cron: 0 8 * * *
```

Comment:
How about we don't run this on the weekend, similar to the [bugwatcher-doctext job above](https://github.com/openshift/release/pull/38053/files#diff-9bd198042384d39bcafc3762f80f03d90910f77e7a701b4bdcd4d6ac39164464R33)?

Reply:
Good point. Perhaps an arbitrary day? Wednesdays?

Reply:
Chose Wednesday